packages/shared/components/Sidebar.svelte

```diff
@@ -75,7 +79,7 @@
                 <Icon width="24" height="24" icon="wallet" />
             </button>
             <button
-                class="{$dashboardRoute === Tabs.Staking ? 'text-blue-500' : 'text-gray-500'} relative"
+                class="mb-8 {$dashboardRoute === Tabs.Staking ? 'text-blue-500' : 'text-gray-500'} relative"
```

Comment:
minor CSS thing, we could simply add `space-y-8` to the `flex` parent of these buttons to avoid adding `mb-8` to each new icon